File: from-a-to-z/fromAToZ.js
const gimmieTheLetters = (string) => {

    const alpha = 'abcdefghijklmnopqrstuvwxyz';
    const start = alpha.indexOf(string.toLowerCase().charAt(0));
    const end = alpha.indexOf(string.toLowerCase().charAt(2)) + 1;

    if(string.charAt(0).toUpperCase() === string.charAt(0)) return alpha.slice(start, end).toUpperCase();
    return alpha.slice(start, end);

};

module.exports = { gimmieTheLetters };


//https://www.codewars.com/kata/6512b3775bf8500baea77663/train/javascript

File: python-venv/from_a_to_z/kata_function/from_a_to_z.py
def gimmie_the_letters(string):
    
    alpha = 'abcdefghijklmnopqrstuvwxyz'
    start = alpha.index(string[0].lower())
    end = alpha.index(string[2].lower()) + 1

    if string[0].upper() == string[0]:
        return alpha[start:end].upper()
    
    return alpha[start:end]

#https://www.codewars.com/kata/6512b3775bf8500baea77663/train/python